scripts/ta_bin_to_c.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2017, Linaro Limited
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+
+def get_args():
+	import argparse
+
+	parser = argparse.ArgumentParser(description='Converts a Trusted '
+		'Application ELF file into a C source file, ready for '
+		'inclusion in the TEE binary as an "early TA".')
+
+	parser.add_argument('--out', required=True,
+		help='Name of the output C file')
+
+	parser.add_argument('--ta', required=True,
+		help='Path to the TA binary. File name has to be: <uuid>.* '
+		'such as: 8aaaf200-2450-11e4-abe2-0002a5d5c51b.stripped.elf')
+
+	parser.add_argument('--compress', dest="compress",
+		action="store_true", help='Compress the TA using the DEFLATE '
+		'algorithm')
+
+	return parser.parse_args()
+
+def main():
+	import array
```

Comment:
I'll change. I also have a slight preference for imports at the top, probably because I'm essentially developing in C ;)